features/bookmarks/bookmark_search.feature

```diff
@@ -241,4 +241,4 @@ Feature: Search Bookmarks
   Scenario: Inputting bad queries
   When I fill in "Any field on work" with "bad~query~~!!!"
     And I press "Search Bookmarks"
-  Then I should see "Your search failed because of a syntax error"
+  Then I should see "No results found. You may want to edit your search to make it less specific."
```

Comment:
This test is meant to check the "Your search failed because of a syntax error" rescue clause, so I don't think it should be updated this way. If ES 7 can handle this query string, let's change to this one used in a work search test which still fails as expected:

https://github.com/otwcode/otwarchive/blob/1f7610f25b28388bd6b2beb4715f2a80ece78ef7/features/search/works_info.feature#L6-L10

Reply:
Thanks for the suggestion. It turns out the underlying query is different on that one and runs against the works index which is why it still fails even though the bookmark one is inexplicably successful (the number of tildes in the query string doesn't make a difference). 

I've done some testing comparing the test index on ES6 (400 as expected) vs the test index on ES7 (200 OK - not expected) vs my local development index on ES7 (400 as expected too). I've summarised the tests in chat so I guess we can discuss more in depth there.

Reply:
(FWIW, it turns out the solution was to populate the index in the test)